assets/javascript/app.js

```diff
@@ -1,190 +1,200 @@
+            choiceHouse.classList = `ml-5 choice-${index} winter btn btn-secondary aria-pressed="true"`;
+            choiceHouse.appendChild(document.createTextNode(choice));
+            questionQC.appendChild(choiceHouse);
+        });
+        // Append questions and choices to the parent div
+        document.querySelector(".questionField").appendChild(questionQC);
+    });
 
+    //creating an array from the winter nodelist 
+    var buttonsArray = Array.from(document.querySelectorAll(".winter"));
+    // for each button added click event and call the check user input function
+    buttonsArray.forEach(button => {
+        button.addEventListener("click", checkUserInput);
     });
-    // Append questions and choices to the parent div
-    document.querySelector(".questionField").appendChild(questionQC);
-});
 
-  //creating an array from the winter nodelist 
-var buttonsArray = Array.from(document.querySelectorAll(".winter"));
-// for each button added click event and call the check user input function
-buttonsArray.forEach(button => {
-    button.addEventListener("click", checkUserInput);
-});
+    function checkUserInput(e) {
+        // capture the event and the traverse the dom to get the question and user selected answer
+        var currentQuestion = $(e.target).first().parent()
+            .attr('class')
+            .split(/\s+/)
+            .filter(function (className) {
+                return className.startsWith('question-')
+            })[0];
+        var questionIndex = currentQuestion[currentQuestion.length - 1];
+
+        var choiceName = $(e.target).first()
+            .attr('class')
+            .split(/\s+/)
+            .filter(function (className) {
+                return className.startsWith('choice')
+            })[0];
+        var userAnswer = choiceName[choiceName.length - 1];
 
-function checkUserInput(e) {
-// capture the event and the traverse the dom to get the question and user selected answer
-    var currentQuestion,
-        userAnswer;
-    if (e.target.classList.contains('choice-0')) {
-        currentQuestion = e.target.previousElementSibling.classList[1];
-        userAnswer = e.target.classList[1];
-        checkCorrectAnswer(currentQuestion, userAnswer);
-    } else if (e.target.classList.contains('choice-1')) {
-        currentQuestion = e.target.previousElementSibling.previousElementSibling.classList[1];
-        userAnswer = e.target.classList[1];
-        checkCorrectAnswer(currentQuestion, userAnswer);
-    } else if (e.target.classList.contains('choice-2')) {
-        currentQuestion = e.target.previousElementSibling.previousElementSibling.previousElementSibling.classList[1];
-        userAnswer = e.target.classList[1];
-        checkCorrectAnswer(currentQuestion, userAnswer);
-    } else if (e.target.classList.contains('choice-3')) {
-        currentQuestion = e.target.previousElementSibling.previousElementSibling.previousElementSibling.previousElementSibling.classList[1];
```

Comment:
This is atrocious lol. So much duplicate code. I suggested a better way of handling this.